src/pages/update-timetable-status-page/UpdateTimetableStatusPage.tsx

```diff
@@ -9,7 +9,13 @@ type UpdateTimetableStatusPageProps = {
 export const UpdateTimetableStatusPage = ({
   errors,
 }: UpdateTimetableStatusPageProps): JSX.Element => {
-  const { setStatusHasChanged } = useFormContext();
+  const { setStatusHasChanged, statusHasChanged } = useFormContext();
+
+  const radioValue = statusHasChanged
+    ? "y"
+    : statusHasChanged == false
+    ? "n"
+    : undefined;
```

Comment:
Could we have set the values for the options to `true` and `false` instead of `"y"` and `"n"`?

Reply:
our Radios component at the moment takes options of the following type:
`{ label: string; value: string };`
so value needs to be a string. we could have "true" and "false" as a strings maybe?